### DISCUSSION TITLE
How to contribute a small pr?

### DISCUSSION THREAD

**Discussion by andrekandore** [italomandara/CXPatcher#98] [General]
hi!

i fixed a small issue with `Info.plist` (its currently lowercase) when searching in `isCrossoverApp()`, and i'd like to send a small pr to fix it so i'm curious what your process for that is?

otherwise i can just put the diff here if you prefer that, its just 1 line ¥^_^¥

## issue
patching failed on a case-sensitive fs because filename is `Info.plist` not `info.plist`
(probably you don't support case-sensitive but its still formally `Info.plist`, so i thought its worth maybe fixing)

## workaround
move crossover.app to a case-insensitive partition/image

## patch
```patch
 func isCrossoverApp(url: URL, version: String? = nil, skipVersionCheck: Bool? = false) -> Bool {
-    let plistPath = url.path + "/Contents/info.plist"
+    let plistPath = url.path + "/Contents/Info.plist"
     if (f.fileExists(atPath: plistPath)) {
```

## tested case
patch completed successfully
run crossover with no problems (3d game)

**Comment by italomandara** [comment]
[you clone the repo create a branch and open a pull request via GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)

**Reply by andrekandore** [reply]
got it, wasn't sure if just sending a pr was ok or prefer discussion/issue first, thanks!!

**Reply by andrekandore** [reply]
pps guess i didnt notice the banner on top of the issues page

```
If you have a bug or an idea, browse the open issues before opening a new one. You can also take a look at the [Open Source Guide](https://opensource.guide/).
```

probably should have opened an issue first, sry

**Reply by andrekandore** [reply]
sent a pr here!

https://github.com/italomandara/CXPatcher/pull/99

**Reply by italomandara** [reply]
What you could do is open an issue and discuss there, then open a PR that references the issue, then close it after PR is merged.

**Reply by andrekandore** [reply]
got it, ill just do that next time, thanks ^^

**Comment by andrekandore** [comment]
@italomandara 
pr was merged, so ill close this discussion, thanks for merging!

**Reply by italomandara** [reply]
Good catch, and thanks for your contribution!
